src/core/algorithms/fd/pyrocommon/model/pli_cache.h

```diff
@@ -29,20 +30,20 @@ class PLICache {
     std::unique_ptr<VerticalMap<PositionListIndex>> index_;
     // usageCounter - for parallelism
 
-    int saved_intersections_ = 0;
+    MAYBE_UNUSED int saved_intersections_ = 0;
 
     mutable std::mutex getting_pli_mutex_;
 
     CachingMethod caching_method_;
-    CacheEvictionMethod eviction_method_;
-    double caching_method_value_;
+    MAYBE_UNUSED CacheEvictionMethod eviction_method_;
+    MAYBE_UNUSED double caching_method_value_;
     // long long maximumAvailableMemory_ = 0;
     double maximum_entropy_;
-    double mean_entropy_;
-    double min_entropy_;
-    double median_entropy_;
-    double median_gini_;
-    double median_inverted_entropy_;
+    MAYBE_UNUSED double mean_entropy_;
+    MAYBE_UNUSED double min_entropy_;
+    MAYBE_UNUSED double median_entropy_;
+    MAYBE_UNUSED double median_gini_;
+    MAYBE_UNUSED double median_inverted_entropy_;
```

Comment:
All these MAYBE_UNUSED variables are required to support Pyro's caching strategies from our ADBIS paper:
https://link.springer.com/chapter/10.1007/978-3-030-30278-8_7

During our meeting that we held an hour ago, we decided to keep them and comment these lines out, while providing this explanation and reference.